benches/benches/vm_set/blockchain.rs

```diff
@@ -517,14 +543,24 @@ pub fn run(c: &mut Criterion) {
         VmBench::new(op::bhei(0x10)),
     );
 
+    let mut db =
+        BenchDb::new(&VmBench::CONTRACT).expect("Unable to fill contract storage");
+    db.add_blocks(10000);
+
     run_group_ref(
         &mut c.benchmark_group("bhsh"),
         "bhsh",
-        VmBench::new(op::bhsh(0x10, RegId::ZERO)).with_prepare_script(vec![
-            op::movi(0x10, Bytes32::LEN.try_into().unwrap()),
-            op::aloc(0x10),
-            op::move_(0x10, RegId::HP),
-        ]),
+        VmBench::new(op::bhsh(RegId::HP, 0x11))
+            .prepend_prepare_script(vec![
+                // Store number of bytes we want to alloc for the future result
+                op::movi(0x10, Bytes32::LEN.try_into().unwrap()),
+                // Allocate space for the future result
+                op::aloc(0x10),
+                // Add block height to 0x11
+                op::movi(0x11, 0xc9),
+            ])
+            .with_height(0xca.into())
```

Comment:
random height chosen here? should it have been the latest height?

Reply:
No it should be any random heihgt doesn't really matter. (and it's better if it's not the latest in case an implementation have a special case for the latest blocks). Added a comment